serverlessworkflow/antora/supplemental-ui/partials/head-scripts.hbs

```diff
@@ -0,0 +1,15 @@
+<!-- Adobe Analytics for Red Hat - DPAL (DTM Property Auto-Loader) - part 1/2 -->
+<script id="dpal" src="https://www.redhat.com/dtm.js" type="text/javascript"></script>
+<!-- Google Analytics for kie team: Global site tag (gtag.js) -->
+{{#with site.keys.googleAnalytics}}
```

Comment:
There is no `googleAnalytics` key provided in the `antora-playbook.yml`.

Reply:
I removed this from the file for now.